### PR TITLE
Create plugin-ecs-publisher.yml

### PR DESCRIPTION
**Pull request by Prizrack13**
https://issues.jenkins-ci.org/browse/HOSTING-386
https://github.com/jenkinsci/ecs-publisher-plugin

### CODE REVIEW

File: permissions/plugin-ecs-publisher.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "ecs-publisher"
+paths:
+- "de/eacg/ecs-publisher"
+developers:
+- "prizrack13"
+- "jthDEV"
```
Comment (daniel-beck) [inline permissions/plugin-ecs-publisher.yml]: No such user exists in Artifactory. Please review the PR checklist and follow its instructions.

Comment (Prizrack13) [reply permissions/plugin-ecs-publisher.yml]: plz check again
I have created jthDEV and login https://repo.jenkins-ci.org/webapp/#/home

Comment (Prizrack13) [reply permissions/plugin-ecs-publisher.yml]: plz check again
I have created jthDEV and login https://repo.jenkins-ci.org/webapp/#/home

**Comment by Prizrack13** [comment]
I have made initial login for two accounts

File: permissions/plugin-ecs-publisher.yml (hunk 1 above)
Comment (daniel-beck) [reply permissions/plugin-ecs-publisher.yml]: Username should be lowercase, as that's how it appears in Artifactory, and this is case-sensitive.

**Pull request merged**